massiveart/husky-validation
# Feature/improvements

Validation:
- Equal
- Regex
- Update related fields (unique)
- Email not check if empty
- HTML5 attributes (min, max, required), (email, url, number, date, time, datetime)
- Set Focus at first error

Mapper:
- Filter for arrays
